Foundation/NSJSONSerialization.swift

```diff
@@ -131,6 +130,14 @@ open class JSONSerialization : NSObject {
             }
         }
         
+        let count = jsonStr.lengthOfBytes(using: .utf8)
+        let bufferLength = count+1 // Allow space for null terminator
+        var utf8: [CChar] = Array<CChar>(repeating: 0, count: bufferLength)
+        if !jsonStr.getCString(&utf8, maxLength: bufferLength, encoding: .utf8) {
+            fatalError("Failed to generate a CString from a String")
```

Comment:
Good question.  The API documentation for Swift String.getCString does not specify what the return value means, but I gather from the implementation that it calls through to NSString.getCString,  the documentation for which says that false would be returned if the buffer is too small, or an encoding error occurs.
In this case, we sized the buffer appropriately, and because UTF8 can represent all Unicode code points, we shouldn't encounter an encoding error.  Perhaps it would be safe to ignore the return value in this case?